index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>Product landing page</title>
+</head>
+<body>
+    <header class="navbar navbar-expand-lg navbar-dark text-white bg-secondary">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">LOGO</a>
+          <button class="nav d-block bg-secondary border-0" type="button" 
+           data-bs-toggle="collapse"
+           data-bs-target="#navmenu"
+           aria-controls="navmenu" 
+           aria-expanded="false" aria-label="Toggle navigation">
+            <span></span>
+            <span></span>
+            <span></span>
+          </button>
+        </div>
+        <div class="collapse navbar-collapse" id="navmenu">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0 align-items-center">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">E-commerce</a></li>
+                <li><a href="#">Examples</a></li>
+                <li><a href="#">Plans</a></li>
+                <li><a href="#">Support</a></li>
+                <li><a href="#">Blog</a></li>
+            </ul>
+        </div>
```

Comment:
I suggest that you would have used the **nav tag** for navbar elements instead of the **div tag**. The nav tag helps screen readers quickly understand the part of the Html they are reading.